.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+.vscode
+node_modules
```

Comment:
Not really. No es un ultra mega big deal, pero hay algunos editores que tienen problemas generando los diffs cuando no hay newline al final, por eso GitHub te los remarca, pero como te decía, no es big deal, así que siéntete libre de ignorar este comentario.